Games/types/Mafia/roles/cards/EnqueueVisitors.js

```diff
@@ -0,0 +1,30 @@
+const Card = require("../../Card");
+const {PRIORITY_VISITORS_ENQUEUE } = require("../../const/Priority");
+
+module.exports = class EnqueueVisitors extends Card{
```

Comment:
[just a note] In future I think enqueueVisitors can be added as a function in risk's library

getVisitors()